cmd/kubeadm/app/phases/upgrade/compute.go

```diff
@@ -60,7 +60,7 @@ func (u *Upgrade) CanUpgradeEtcd() bool {
 type ClusterState struct {
 	// KubeVersion describes the version of the Kubernetes API Server, Controller Manager, Scheduler and Proxy.
 	KubeVersion string
-	// DNSType
+	// DNSType the DNS add-on to be used.
```

Comment:
Thanks @xichengliudui 
But would that be a better description?
```suggestion
	// DNSType describes the type of DNS add-on used in the cluster
```

Reply:
/cc @neolit123 
/cc @SataQiu

Reply:
yes, `DNSType describes the type of DNS add-on used in the cluster` is better.